README.md

```diff
@@ -45,49 +31,50 @@ All new features should be **secondary features**, so you need to write cleanup
 
 [**Standard testsuite features Features included in the `testsuite.yml` file will be executed sequentially from the top to the bottom**](https://github.com/SUSE/spacewalk-testsuite-base/blob/master/run_sets/testsuite.yml).
 
-## Run just a single run set
-
-You can override the tests that are executed by adding your run set:
-1. Login into the `controller` host.
-2. Add the feature(s) you would like to run into `run_sets/$name.yml`
-3. Call Cucumber to execute your custom run set (`rake cucumber:$name`)
-
 
 ## Images used on the testsuite.
 
 The images are build by kiwi : http://download.suse.de/ibs/Devel:/Galaxy:/Terraform:/Images/, and sumaform deploy them.
 
-
 # Contributing
 
 ## Procedure
 
-1. Fork it
-2. Create your branch containing the new tests, starting from master (`git checkout master && git checkout -b my-new-tests`)
-3. Commit your changes (`git commit -am 'Add some tests'`)
-4. Push to the branch (`git push origin my-new-tests`)
-5. Create new Pull Request
-6. **IMPORTANT**: Create a Pull Request to backport your new tests to the other branches used in this project.
+1. Create **always** PR. (even for backporting)
+2. Your PR need always at least Reviewer that approve.
```

Comment:
I would rephrase to: "Your PR needs always at least one Reviewer that approves".